tools/klee/main.cpp

```diff
@@ -249,6 +266,58 @@ class KleeHandler : public InterpreterHandler {
   unsigned getNumPathsExplored() { return m_pathsExplored; }
   void incPathsExplored() { m_pathsExplored++; }
 
+  void incTotalDepthPathsExploredOnExit(unsigned currentDepth) {
+    m_totalDepthPathsExploredOnExit += currentDepth;
+  }
+
+  void incTotalInstructionsOnExit(unsigned totalInstruction) {
```

Comment:
`totalInstruction` -> `instructionsDepth`. I suggest renaming this argument in all similar functions.